src/library_webgpu.js

```diff
@@ -2008,7 +2008,9 @@ var LibraryWebGPU = {
 
   wgpuTextureGetFormat: function(textureId) {
     var texture = WebGPU.mgrTexture.get(textureId);
-    return texture.format;
+    // TODO(#18962): use reverse-lookup table.
```

Comment:
Actually I don't think we'll even want a reverse-lookup table, it will bloat the build size for very little benefit.
```suggestion
```